configure.ac

```diff
@@ -6307,7 +6307,9 @@ fi
       echo "BCL_OPTIMIZE = 1" >> $srcdir/$mcsdir/build/config.make
     fi
 
-    echo "CSC_LOCATION = $CSC_LOCATION" >> $srcdir/$mcsdir/build/config.make
+    echo "STANDALONE_CSC_LOCATION=$CSC_LOCATION" >> $srcdir/$mcsdir/build/config.make
```

Comment:
@kg This looks like the culprit for the Linux WebAssembly build.

The `CSC_LOCATION` is being used in the wasm build.

https://github.com/mono/mono/blob/master/sdks/wasm/Makefile#L22

Reply:
Thanks.